docs/modules/modules.md

```diff
@@ -0,0 +1,91 @@
+---
+order: false
+---
+
+# Auth 
+
+The `x/auth` modules is used for accounts
+
+See the [API docs](https://godoc.org/github.com/cosmos/cosmos-sdk/x/auth)
+
+See the [specification](https://github.com/cosmos/cosmos-sdk/tree/master/docs/spec/auth)
+
+# Bank
+
+The `x/bank` module is for transferring coins between accounts.
+
+See the [API docs](https://godoc.org/github.com/cosmos/cosmos-sdk/x/bank).
+
+See the [specification](https://github.com/cosmos/cosmos-sdk/tree/master/docs/spec/bank)
+
+# Stake
+
+The `x/staking` module is for Cosmos Delegated-Proof-of-Stake.
+
+See the [API docs](https://godoc.org/github.com/cosmos/cosmos-sdk/x/staking).
+
+See the
+[specification](https://github.com/cosmos/cosmos-sdk/tree/master/docs/spec/staking)
+
+# Slashing
+
+The `x/slashing` module is for Cosmos Delegated-Proof-of-Stake.
+
+See the [API docs](https://godoc.org/github.com/cosmos/cosmos-sdk/x/slashing)
+
+See the
+[specification](https://github.com/cosmos/cosmos-sdk/tree/master/docs/spec/slashing)
+
+# Distribution
+
+The `x/distribution` module is for distributing fees and inflation across bonded
+stakeholders.
+
+See the [API docs](https://godoc.org/github.com/cosmos/cosmos-sdk/x/distribution)
+
+See the
+[specification](https://github.com/cosmos/cosmos-sdk/tree/master/docs/spec/distribution)
+
+# Governance
+
+The `x/gov` module is for bonded stakeholders to make proposals and vote on them.
+
+See the [API docs](https://godoc.org/github.com/cosmos/cosmos-sdk/x/gov)
+
+See the
+[specification](https://github.com/cosmos/cosmos-sdk/tree/master/docs/spec/governance)
+
+# IBC
+
+The `x/ibc` module is for InterBlockchain Communication.
+
+See the [API docs](https://godoc.org/github.com/cosmos/cosmos-sdk/x/ibc)
+
+See the
+[specification](https://github.com/cosmos/cosmos-sdk/tree/master/docs/spec/ibc)
+
+To keep up with the current status of IBC, follow and contribute to [ICS](https://github.com/cosmos/ics)
+
+# Crisis
+
+The `x/crisis` module is for halting the blockchain under certain circumstances.
+
+See the [API Docs](https://godoc.org/github.com/cosmos/cosmos-sdk/x/crisis)
+
+See the [specification](https://github.com/cosmos/cosmos-sdk/blob/master/docs/spec/crisis)
+
+# Mint
+
+The `x/mint` module is for flexible inflation rates and effect a balance between market liquidity and staked supply.
+
+See the [API Docs](https://godoc.org/github.com/cosmos/cosmos-sdk/x/mint)
+
+See the [specification](https://github.com/cosmos/cosmos-sdk/blob/master/docs/spec/mint)
+
+# Params
```

Comment:
this is missing the `Supply` module

Reply:
This branch consists of what @gamarin2 merged a couple of weeks ago plus all the changes to `master` that I've manually copied. Some things may be missing.

Reply:
can you create an issue to track this work?